docs/source/reference/frame.rst

```diff
@@ -113,3 +113,4 @@ Serialization / IO / Conversion
    :toctree: api/
 
    DataFrame.to_html
+   DataFrame.to_numpy
```

Comment:
Although pandas doesn't have `to_numpy` in `DataFrame`, I think we should have this in the same section as our `Series`, "Conversion" section.